Fabien branch front

Respeccs opened the pull request:
Let me know if improvements are needed.

TheRealPlagiariser commented:
Cuma mo rente lakaz mo chk li, add a description in regards to what you have done, I've linked the pull req with the current issue, Also you do not have to drag it from "in progress" to "done", this is automated once we do the merge.

Respeccs commented:
- Added getClaimByUserID() in user,service.ts
- Subscribed to getClaimByUserID() in viewclaim,component.ts
- Added session storage in employee.component,ts and user.service.ts

TheRealPlagiariser commented on src/app/services/users/users.service.ts:
```diff
@@ -12,6 +13,17 @@ export class UsersService {
+
+  getClaimByUserID(UserData): Observable<any>{
+    let empid = sessionStorage.getItem('employeeId')
+    return this.http.post(`https://localhost:44382/GetClaimsById?employeeId=${empid}`, {UserData});
```
use environment declared in environment.ts, do not put api link here
(`${environment.apiUrl}/

TheRealPlagiariser replied:
use environment declared in environment.ts, do not put api link here
(`${environment.apiUrl}/

TheRealPlagiariser reviewed (commented):
Once you have changed the link to environment, let me know so we can merge asap
